frontends/PyCDE/test/test_hwarith.py

```diff
@@ -75,16 +83,24 @@ def construct(ports):
 # CHECK: msft.module @InfixComparison {} (%in0: ui16, %in1: ui16)
 # CHECK-NEXT:    %0 = hwarith.icmp eq %in0, %in1 {sv.namehint = "in0_eq_in1"} : ui16, ui16
 # CHECK-NEXT:    %1 = hwarith.icmp ne %in0, %in1 {sv.namehint = "in0_neq_in1"} : ui16, ui16
+# CHECK-NEXT:    %1 = hwarith.icmp lt %in0, %in1 {sv.namehint = "in0_lt_in1"} : ui16, ui16
+# CHECK-NEXT:    %1 = hwarith.icmp gt %in0, %in1 {sv.namehint = "in0_gt_in1"} : ui16, ui16
+# CHECK-NEXT:    %1 = hwarith.icmp le %in0, %in1 {sv.namehint = "in0_le_in1"} : ui16, ui16
+# CHECK-NEXT:    %1 = hwarith.icmp ge %in0, %in1 {sv.namehint = "in0_ge_in1"} : ui16, ui16
 # CHECK-NEXT:    msft.output
 @unittestmodule(run_passes=False)
 class InfixComparison(Module):
-  in0 = Input(types.ui16)
-  in1 = Input(types.ui16)
+  in0 = Input(UInt(16))
```

Comment:
This is the correct place to add the new tests. Just testing UInt should be sufficient since they generate the same IR.